src/ray/gcs/gcs_client/gcs_client.h

```diff
@@ -39,11 +40,11 @@ namespace gcs {
 /// password.
 class GcsClientOptions {
  public:
-  /// Constructor of GcsClientOptions.
+  /// Constructor of GcsClientOptions from redis.
   ///
-  /// \param ip GCS service ip.
-  /// \param port GCS service port.
-  /// \param password GCS service password.
+  /// \param ip redis service ip.
```

Comment:
Since we are fixing these lines, how about also fixing the variable names, `redis_ip` and `redis_port`?

Reply:
I think it doesn't matter. I'd prefer not to fix them since we are removing them in near future anyway.